src/libraries/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -1289,6 +1290,57 @@ public void ConcurrentGetProperties_ReturnsExpected()
             }
         }
 
+        [SkipOnPlatform(TestPlatforms.Browser, "Thread.Start is not supported on browsers.")]
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public static void ConcurrentAddProviderAndGetProvider()
+        {
+            // Use a timeout value lower than RemoteExecutor in order to get a nice Fail message.
+            const int Timeout = 50000;
+
+            RemoteInvokeOptions options = new()
+            {
+                TimeOut = 60000
+            };
+
+            ConcurrentAddProvider();
```

Comment:
You call this in the test process, which ends up registering `EmptyPropertiesTypeProvider` for `typeof(MyClass)`, this breaks the remaining tests (GetConverter_ByMultithread_ReturnsExpected) which run in process and expect the provider registered with `TypeDescriptionProviderAttribute`.

What made you call `ConcurrentAddProvider` in proc?  Seems like the rest of the test is happening out of proc.